src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleDeclarations = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.length > 0);
+
+  return styleDeclarations.reduce((acc, style) => {
+    const [key, value] = style.split(':');
```

Comment:
[CODE KNOWLEDGE] - Since you are not handling cases with multiple colons within the style string's property values, this implementation is acceptable. However, be aware that in a real-world scenario, CSS property values could contain colons, such as in URLs or content attributes, which would require additional parsing logic.